runtime/ibm_vpc/build_lithops_vm_image.sh

```diff
@@ -76,15 +76,53 @@ virt-customize -a ubuntu-20.04-server-cloudimg-amd64.img  --run-command 'rm -rf
 virt-customize -a ubuntu-20.04-server-cloudimg-amd64.img  --run-command 'apt-cache search linux-headers-generic'
 printf "\n\n"
 
+
+include_docker(){
+    # Include docker image
+    echo "-------------------------------------------"
+    echo "--> Including docker image into the VM  <--"
+    echo "-------------------------------------------"
+    echo "Docker image: $DOCKER_IMAGE"
+    echo ""
+    # docker system prune -a
```

Comment:
@JosepSampe without this line, it will tar entire docker folder which might be tons on MBs?

Reply:
@gilv I think this is a critical command. There is no poblem if you run the script in VM specially created only for running this script. But if some user runs the script in his ubuntu laptop, it will delete all his images, so I don't know, uncomment it if you prefer.